proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -89,6 +89,8 @@ message Status {
   string status_message = 2;
   int64 lastUpdate_timestamp = 3;
```

Comment:
```suggestion
  int64 updated_at = 3;
```

Reply:
That is a breaking change which would break our other tooling in the indexer and monitoring. I don't want to make breaking changes unless there is a significant value being added.

Reply:
Makes sense